idd/Energy+.idd.in

```diff
@@ -7604,6 +7604,54 @@ MaterialProperty:VariableThermalConductivity,
        \units W/m-K
        \type real
 
+MaterialProperty:VariableThermalAbsorptance,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \note The name of the dynamic coating material with variable thermal absorptance.
+  A2 , \field Reference Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialName
+       \note Regular Material Name to which the additional properties will be added.
+       \note this the material name for the basic material properties.
+  A3 , \field Control Signal
+       \required-field
+       \type choice
+       \key SurfaceTemperature
+       \key SurfaceReceivedSolarRadiation
+       \key SpaceHeatingCoolingMode
+       \key Scheduled
+       \default SurfaceTemperature
+  A4 , \field Trigger Thermal Absorptance Function Name
+       \note A Curve:* or Table:Lookup object encoding the relationship between the trigger and surface thermal absorptance.
+  A5 ; \field Thermal Absorptance Schedule Name
+       \note only used when Control Signal = "Scheduled"
```

Comment:
Do these materials commonly have both variable thermal and variable solar absorptance or just or the other? I'm wondering if these should be combined into a single object... 🤔